scripts/passGenerator.py

```diff
@@ -0,0 +1,34 @@
+import random
+import string
+
+def generate_password(length):
```

Comment:
Minor: Could you add a docstring that explains what the function does (I know it is pretty self explanatory from reading its' name, but it is just for the sake of consistency with the other scripts in the repo)

For example:

`
def generate_password(length):
    """Generates the password.
    
        Args: length (int), the length of the password

        Returns: password (str), the password in string format
`

Reply:
sure